src/sentry/static/sentry/app/routes.jsx

```diff
@@ -391,12 +391,14 @@ const projectSettingsRoutes = (
         component={errorHandler(LazyLoad)}
       />
     </Route>
-    <Route path="plugins/" name="Integrations" component={errorHandler(ProjectPlugins)} />
-    <Route
-      path="plugins/:pluginId/"
-      name="Integration Details"
-      component={errorHandler(ProjectPluginDetails)}
-    />
+    <Route key="plugins/" path="plugins/" name="Integrations">
```

Comment:
I don't think you need `key`, it's not an array

Reply:
The route above it had key, so I was just being consistent.

Reply:
Ah, well those don't need to have `key`